docs/content/preview/api/ysql/the-sql-language/statements/dml_fetch.md

```diff
@@ -53,20 +53,23 @@ A _cursor_ represents the current position in its result set. After declaring a
 - The `FETCH FORWARD 0` variant fetches the row at the current position and leaves the current position unchanged.
 - The `FETCH NEXT` variant, the bare `FETCH` variant, the bare `FETCH FORWARD` variant, and the `FETCH FORWARD 1` variant all fetch the row immediately after the current position and update the current position to the just-fetched row. However, if before executing one of these `FETCH` variants, the current position is the last row in the result set, then the `FETCH` runs off the end of the available rows, an empty result is returned, and the cursor position is left _after_ the last row. There are no flavors of _after the last row_. It's a uniquely defined state so that following any number of invocations of `FETCH NEXT` in this state, `FETCH PRIOR` will then fetch the last row in the result set (and update the current position to that last row.)
 
-- The `FETCH PRIOR` variant, the bare `FETCH BACKWARD` variant, and the `FETCH BACKWARD 1` variant all fetch the row immediately before the current position and update the current position to the just-fetched row. However, if before executing one of these `FETCH` variants, the current position is the first row in the result set, then the `FETCH` runs off the start of the available rows, an empty result is returned, and the cursor position is left _before_ the first row. There are no flavors of _before the first row_. It's a uniquely defined state so that after following any number of invocations of `FETCH PRIOR` in this state, `FETCH NEXT` will then fetch the first row in the result set (and update the current position to that first row.)
+- The `FETCH PRIOR` variant, the bare `FETCH BACKWARD` variant, and the `FETCH BACKWARD 1` variant all fetch the row immediately before the current position and update the current position to the just-fetched row. However, if before executing one of these `FETCH` variants, the current position is the first row in the result set, then the `FETCH` runs off the start of the available rows, an empty result is returned, and the cursor position is left _before_ the first row. There are no flavors of _before the first row_. It's a uniquely defined state so that after following any number of invocations of `FETCH PRIOR` in this state, `FETCH NEXT` will then fetch the first row in the result set (and update the current position to that first row.)\*
```

Comment:
```suggestion
- The `FETCH PRIOR` variant, the bare `FETCH BACKWARD` variant, and the `FETCH BACKWARD 1` variant all fetch the row immediately before the current position and update the current position to the just-fetched row. However, if before executing one of these `FETCH` variants, the current position is the first row in the result set, then the `FETCH` runs off the start of the available rows, an empty result is returned, and the cursor position is left _before_ the first row. There are no flavors of _before the first row_. It's a uniquely defined state so that after following any number of invocations of `FETCH PRIOR` in this state, `FETCH NEXT` will then fetch the first row in the result set (and update the current position to that first row).\*
```

Reply:
Good catch. Will fix.